proof/drefine/Tcb_DR.thy

```diff
@@ -924,27 +933,6 @@ lemma dcorres_tcb_empty_slot:
   apply (clarsimp simp:cte_wp_at_caps_of_state)
   done
 
-lemma dcorres_idempotent_as_user_strong:
-  assumes prem: "\<And>tcb ms ref P.
-                 \<lbrace> \<lambda>cxt. P (transform_tcb ms ref (tcb\<lparr>tcb_arch:=arch_tcb_context_set cxt (tcb_arch tcb)\<rparr>))\<rbrace>
-                 x
-                 \<lbrace> \<lambda>_ cxt. P (transform_tcb ms ref (tcb\<lparr>tcb_arch:=arch_tcb_context_set cxt (tcb_arch tcb)\<rparr>))\<rbrace>"
-  shows "dcorres dc \<top> (tcb_at u) (return q) (as_user u x)"
```

Comment:
Out of curiosity, why can this now be removed? Was it not being used?

Reply:
Yep, not used at all